src/build/help/help.xml

```diff
@@ -2246,6 +2246,22 @@
                 </text>
 
                 <option-list>
+                    <option id="oldest" name="Oldest">
+                        <summary>Expire the oldest eligible backup set.</summary>
+
+                        <text>
+                            <p>Expire the oldest full backup chain that can be removed (meaning at least one newer full backup remains). This is equivalent to manually decrementing retention by one, but computed automatically.</p>
+
```

Comment:
Let's use 'set' consistently instead of introducing new terminology like 'chain'. This applies in some of the code comments as well.